README

```diff
@@ -24,12 +24,13 @@ not part of the NDK) and a number of other apps/programs.
 Project links:
 
   * [API documentation](https://commondatastorage.googleapis.com/chromium-boringssl-docs/headers.html)
-  * [Bug tracker](https://issues.chromium.org/issues?q=status:open%20componentid:1590116%2B&s=created_time:desc)
-    * [Filing new issues](https://issues.chromium.org/issues/new?component=1590116&template=2003314)
-    * [Filing new security issues](https://issues.chromium.org/issues/new?component=1590116&template=2003205)
+  * [Issue tracker](https://crbug.com/boringssl)
+    * [Filing new (public) issues](https://crbug.com/boringssl/new)
   * [CI](https://ci.chromium.org/p/boringssl/g/main/console)
   * [Code review](https://boringssl-review.googlesource.com)
 
+To file a security issue, use the [Chromium process](https://www.chromium.org/Home/chromium-security/reporting-security-bugs/) and mention in the report this is for BoringSSL. You can ignore the parts of the process that are specific to Chromium/Chrome.
```

Comment:
Yes, that's the original README, so we should keep it unchanged. In addition, if any issues arise, they could be due to Google rather than our modifications. Therefore, such problems should be reported to Google, not to us.

Reply:
> they could be due to Google rather than our modifications. 

They could be. But they also could be in "OQS turf". Anyway, we don't give any, particular security incident, support on this sub project -- plus there's a clear "do not use me really" warning, so this is fine then.